src/mod_muc_api.erl

```diff
@@ -115,7 +114,9 @@ create_instant_room(MUCDomain, Name, OwnerJID, Nick) ->
                 Error ->
                     Error
             end;
-        false ->
+        {true, error} ->
+            {invalid_input, "Room name or domain are invalid. Check the format of the arguments."};
```

Comment:
I would change the error message to "Room name or domain are invalid" to keep it consistent with other messages (no dot, one sentence).

Reply:
ok